test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -76,6 +71,65 @@ public class RequestTests
             }
         }
 
+        [ConditionalFact]
+        [FrameworkSkipCondition(RuntimeFrameworks.Mono, SkipReason = "Needs to run on 64-bit Mono, which isn't available on OSX.")]
```

Comment:
Sorry if I'm missing something obvious, but why does this need to run on 64-bit mono? `long` is the same size on 32-bit mono right? Do you run out of address space or something?